coordinator/src/node.rs

```diff
@@ -95,15 +91,15 @@ impl Node {
     }
 
     async fn open_position(&self, trade_params: &TradeParams) -> Result<()> {
-        tracing::info!("Opening position");
+        tracing::info!(?trade_params, "Opening position");
```

Comment:
Can we have the peer id in here somehow?

Reply:
trade params has pubkey, hope it's good for now.